modules/nicknames/events/botReady.js

```diff
@@ -0,0 +1,9 @@
+const {renameMember} = require('../renameMember');
+
+module.exports.run = async function (client) {
+    console.log("EEEE")
+    for (const member of client.guild.members.cache) {
```

Comment:
```suggestion
    for (const member of client.guild.members.cache.values()) {
```
The cache is a discord.js Collection, requiring the use of .values() to fix your issue